src/components/ng-redux.ts

```diff
@@ -52,11 +58,20 @@ export class NgRedux<RootState> implements ObservableStore<RootState> {
    * @param enhancers Optional Redux store enhancers
    */
   configureStore(
-    rootReducer: Reducer<RootState>,
-    initState: RootState,
+    rootReducer?: Reducer<RootState>,
+    initState?: RootState,
```

Comment:
I'm probably out of context, but I don't get why these arguments needs to be optional